### ISSUE TITLE
Fix PageSpeed Insights

### ISSUE THREAD

**Issue by hmitsch** [mozilla/coss#488]
## Google PageSpeed Insights
https://developers.google.com/speed/pagespeed/insights/?url=opensource.mozilla.community&tab=mobile

## Issues Reported
* Optimize CSS Delivery 
* Enable gzip compression
* Leverage browser caching: Setting an expiry date or a maximum age in the HTTP headers for static resources
* Minify HTML

**Comment by akatsoulas** [comment]
@comzeradd did you have a chance to review this at all?

**Comment by comzeradd** [comment]
Yes, I checked most of the issues in this report. The suggestions expand to various levels, with most of them being on the infrastructure side of things. Should we convert this issue to a checklist or open separate issues?

**Comment by akatsoulas** [comment]
We can do some of them in the application (Django) level. That said, I believe the biggest issue is that we are using raw images without any manipulation which results in images of really big size. Can you have a look at this?

**Comment by comzeradd** [comment]
Yes, definitely. Wagtail provides some nice image manipulation helpers. I can work on this.
